Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -156,7 +157,11 @@ public static void FillComboFromEnumObj(ComboBox comboBox, Object EnumObj, List<
                     ComboEnumItem CEI = new ComboEnumItem();
                     CEI.text = GetEnumValueDescription(Etype, item);
                     CEI.Value = item;
-                    comboBox.Items.Add(CEI);
+                    if(CEI.text != "NA" && CEI.text != "VB Script" && CEI.text != "Ginger Service" && CEI.text != "ASCF")
```

Comment:
We have option to send excludelist in the function, try adding this values in exclude list so it will not get added